packages/web-monetization-wext/src/content/DocumentMonetization.ts

```diff
@@ -99,15 +105,10 @@ export class DocumentMonetization {
   }
 
   postMonetizationStartWindowMessageAndSetMonetizationState(
```

Comment:
Should we just remove this method and have its callers call `setState` directly?

Reply:
Yeah, could/should inline it eh

I'll do it, but feel free, it's your PR! :)